zend_abstract_interface/tsrmls_cache.h

```diff
@@ -2,13 +2,13 @@
 
 #include <php.h>
 
-#if PHP_VERSION_ID >= 80000 && PHP_VERSION_ID < 80101 && defined(ZTS)
+#if PHP_VERSION_ID >= 80000 && PHP_VERSION_ID < 80100 && defined(ZTS)
```

Comment:
Currently, we would run the code for PHP 8.1.0, which is different with this change. I don't know all the implications of this change; I want to be sure we're not breaking something that is untested, and that could be tricky to debug down the road

Reply:
It doesn't work like that, what's relevant is the PHP_VERSION_ID against which the code is compiled.
The output must be the same regardless of the patch version used.